Ollivanders/src/net/pottercraft/Ollivanders2/Book/Book.java

```diff
@@ -58,9 +59,9 @@
     */
    public Book ()
    {
-      author = new String ("Azami7");
-      title = new String ("Untitled");
-      shortTitle = new String ("Untitled");
+      author = "Azami7";
```

Comment:
I thought we were changing this to "Unknown"

Reply:
Oh yea, forgot to do that.